integrations/extensions/docs/elasticsearch-install-and-setup/how_to_use_web_crawler_in_elasticsearch.md

```diff
@@ -0,0 +1,271 @@
+# Working with web crawler in Elasticsearch
+This is a documentation about how to set up and use web crawler in Elasticsearch and connect it to watsonx Assistant for Conversational Search.
+
+## Tabel of contents:
+* [Step 1: Set up Enterprise Search to enable web crawler in Elasticsearch](#step-1-set-up-enterprise-search-to-enable-web-crawler-in-elasticsearch)
+* [Step 2: Create and configure a web crawler in Elasticsearch](#step-2-create-and-configure-a-web-crawler-in-elasticsearch)
+* [Step 3: Build an ELSER ingest pipeline with a chunking processor](#step-3-build-an-elser-ingest-pipeline-with-a-chunking-processor)
+* [Step 4: Connect a web crawler index to watsonx Assistant for conversational search](#step-4-connect-a-web-crawler-index-to-watsonx-assistant-for-conversational-search)
+
+## Step 1: Set up Enterprise Search to enable web crawler in Elasticsearch
+Before you start, you will need to install and set up your Elasticsearch cluster,
+* For Elasticsearch on IBM Cloud, please refer to [ICD-elasticsearch-install-and-setup](../../docs/elasticsearch-install-and-setup/ICD_Elasticsearch_install_and_setup.md) for more details.
+* For Elasticsearch (watsonx Discovery) on CloudPak, please refer to [watsonx-discovery-install-and-setup](../../docs/elasticsearch-install-and-setup/watsonx_discovery_install_and_setup.md) for more details.
+
+### Set up Enterprise Search for Elasticsearch on IBM Cloud 
+Assuming you have installed Kibana locally following [ICD-elasticsearch-install-and-setup](../../docs/elasticsearch-install-and-setup/ICD_Elasticsearch_install_and_setup.md), 
+follow these steps to set up Enterprise Search in Elasticsearch:
+* Create a docker network
+    ```shell
+    docker network create elastic
+    ```
+  NOTE: `elastic` will be the name of your docker network.
+
+
+* Restart Kibana within a network and with enterprise search host
+  ```shell
+  docker run -it --name kibana --network elastic --rm \
+  -v <path_to_your_kibana_config_folder>:/usr/share/kibana/config \
```

Comment:
Eventually, it may also be better to coalesce the startup of kibana and enterprise search into a `YAML` file that can be started with docker-compose. It may avoid duplication of the startup command in multiple places and for maintainability over time in case we want to update it in the future